servers/parser-graphql-wrapper/src/apollo/resolvers.ts

```diff
@@ -93,32 +84,20 @@ export const resolvers: Resolvers = {
       //       // (e.g. via refreshArticle mutation), otherwise load the article
       //       const article =
       //         parent.parsedArticle ??
-      const article = await (
-        dataSources.parserAPI as ParserAPI
-      ).articleLoader.load({
-        url: parent.givenUrl,
-        options: {
-          imageStyle: MediaTypeParam.AS_COMMENTS,
-          maxAge: info.cacheControl.cacheHint.maxAge,
-        },
+      const article = await dataSources.parserAPI.getItemData(parent.givenUrl, {
+        videos: MediaTypeParam.AS_COMMENTS,
+        images: MediaTypeParam.AS_COMMENTS,
       });
       // Only extract Marticle data from article string if Parser
       // extracted valid data (isArticle or isVideo is 1)
-      return article.isArticle || article.isVideo
+      return article.isArticle || article.hasVideo == Videoness.HasVideos
```

Comment:
probably this should check isVide or hasVideos, not just one

Reply:
So the parserAPI parses into our Item object which doesnt have isVideo. I'll make that take this into account